assets/css/style.css

```diff
@@ -408,6 +408,10 @@ html[light-mode="dark"] {
         width: 50%;
         
     }
+    
+    .no-underline{
```

Comment:
add` !important` keyword then it will work otherwise in deploy preview it will not show